packages/components/src/Text/Text.tsx

```diff
@@ -127,16 +101,15 @@ export type TypographyProps = {
    */
   color?: TypographyColor;
   /**
-   * The size of the html element. If no `as` prop is provided, then
-   * the component uses this value to determine which html tag to render
+   * The size of the html element. If used as a Heading and no `as` prop is provided,
```

Comment:
Should we remove the docs here about using as a Heading component, since there is a Heading component now?